setup.py

```diff
@@ -1,11 +1,22 @@
+from io import open
+from os import path
+
 from setuptools import setup, find_packages
 
 from fetchai.ledger import __version__ as version_string
 
+project_root = path.dirname(__file__)
+
+with open(path.join(project_root, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
 setup(
     name='fetchai-ledger-api',
     version=version_string,
     description='Tools and utilities for interacting with the ledger on the fetch network',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
     url='https://github.com/fetchai/ledger-api-py',
     author='Edward FitzGerald',
     author_email='edward.fitzgerald@fetch.ai',
```

Comment:
Shouldn't packages below exclude `examples`?
Why are we including some files using the manifest, but some using `package_data` in this file? In other words, what is the difference?